docs/self-hosted/config-options.md

```diff
@@ -236,8 +236,8 @@ prefixing the value with `{type}:`. The following types are available:
 | `LOG_LEVEL`                | What level of detail to log. One of `fatal`, `error`, `warn`, `info`, `debug`, `trace` or `silent`.        | `info`                       |
 | `LOG_STYLE`                | Render the logs human readable (pretty) or as JSON. One of `pretty`, `raw`.                                | `pretty`                     |
 | `MAX_PAYLOAD_SIZE`         | Controls the maximum request body size. Accepts number of bytes, or human readable string.                 | `1mb`                        |
-| `ROOT_REDIRECT`            | Where to redirect to when navigating to `/`. Accepts a relative path, absolute URL, or `false` to disable. | `./admin`                    |
-| `SERVE_APP`                | Whether or not to serve the Admin App under `/admin`.                                                      | `true`                       |
+| `ROOT_REDIRECT`            | Redirect the root of the application `/` to a specific route. Accepts a relative path, absolute URL, or `false` to disable. | `./admin`                    |
```

Comment:
IMO, what would really help here is a description of what the default purpose of this redirect is — the redirection from `/` to `/admin`, so that the user can easily reach the app via `https://directus.example.com/` instead of having to remember and type in `https://directus.example.com/admin`.

Of course in a better phrasing 😇

Reply:
Ah no this is just an excerpt of the diff, since I've split my feedback into two separate comments. See https://github.com/directus/directus/pull/19725#discussion_r1328878682.